plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/Activator.java

```diff
@@ -27,7 +27,7 @@
     @Override
     public void preferenceChange(PreferenceChangeEvent event) {
       Object newValue = event.getNewValue();
-      if (PreferenceConstants.CLOUDSDK_PATH.equals(event.getKey())
+      if (PreferenceConstants.CLOUD_SDK_PATH.equals(event.getKey())
```

Comment:
Does this need to check that the `CloudSdkManagement` option is `CUSTOM`?  We should probably retain the preference if the user switches to `MANAGED` should they switch back again.

Reply:
I think we don't need to check the option here. We retain the SDK location value regardless of the managed option, i.e., switching back and forth won't fire the change anyway.

Reply:
I think this does need tweaking.  The purpose of this code is so that code can use the E4 DI to do:
```
@Inject CloudSdk sdk;
```
But I introduced this before we had the pluggable path resolver, and most uses will need to use the `CloudSdk.Builder` to add listeners, etc.  We should tear this out.

Reply:
Stripping out CloudSdk injection: #2751